### DISCUSSION TITLE
Ignore pages

### DISCUSSION THREAD

**Discussion by Willi-Coyote** [red6/pdfcompare#115] [Q&A]
Hi, 
I want to compare 2 PDFs. One has an additional page which should be completely ignored (as it would not be there at all). Here is an example:

PDF 1-----------------------PDF2
1 ----------compareTo--->1
2 (additional)  ----> ignored                 
3----------compareTo---> 2

So the main thing here is that page 3 of PDF 1 will be compared to page 2 of PDF 2

Thanks 
Thomas

**Comment by finsterwalder** [comment]
This is currently not possible.
You can compare page 1 to page one and ignore the rest.
But you can not "cut out" a page in one document, but not the other one.
I would try to manipulate PDF 1 by cutting out the page I want to ignore and then do the comparison on the modified PDF document.
This could be done by PdfBox I'm sure.
I currently have no plan to add this feature to PdfCompare.
